app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -207,6 +209,26 @@
     this.setDimension(21, position);
   };
 
+  StaticAnalytics.prototype.setTotalNumberOfSections = function() {
+    var sidebarSections = $('.govuk-related-items h2').length;
```

Comment:
Do you mean do something similar to here: https://github.com/alphagov/static/pull/996/files#diff-a15d624c56e33899934804a2828f0d7fR222 ?